domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -3246,9 +3280,24 @@ public void selectSingle(AsyncResult<SQLConnection> conn, String sql, Handler<As
    * @param replyHandler  The query result or the failure.
    */
   public void selectSingle(String sql, Tuple params, Handler<AsyncResult<Row>> replyHandler) {
+    getSQLConnection(conn -> selectSingle(conn, sql, params, closeAndHandleResult(conn, replyHandler)));
+  }
+
+  /**
+   * Run a parameterized/prepared select query with the readonly connection
+   * and return the first record, or null if there is no result.
+   *
+   * <p>To update see {@link #execute(String, Handler)}.
```

Comment:
This line is misleading. It should be removed and a new line been appended to the javadoc:

```
   * @see #selectSingle(String, Tuple, Handler)
```